Makefile

```diff
@@ -31,3 +31,21 @@ tag:
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
 	docker build -t ${IMG} .
+
+sbom-tool:
+	curl -Lo sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
+	chmod +x sbom-tool
+
+.PHONY: sbom
+sbom: sbom-tool
+	./sbom-tool generate \
+		-D true \
+		-V Verbose \
+		-b ./target/release \
+		-bc . \
+		-m . \
+		-di ${IMG} \
+		-nsb https://kubewarden.io \
+		-nsu policy-server \
+		-pn policy-server \
```

Comment:
I think we should add `-ps` (package supplier), set to Kubewarden or `https://kubewarden.io.

Reply:
This is an option not released it: https://github.com/microsoft/sbom-tool/commit/66fe849ce701998264da7f9cd731f937500589eb

Reply:
ooh, then let's skip it.

Reply:
It seems they have just released a new version, and it needs `-ps kubewarden` now, or:
`Encountered error while running ManifestTool generation workflow. Error: The value of PackageSupplier can't be null or empty.`